vips/vip-356/bscmainnet.ts

```diff
@@ -0,0 +1,102 @@
+import { ethers } from "hardhat";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { LzChainId, ProposalType } from "src/types";
+import { makeProposal } from "src/utils";
+
+const { arbitrumone, ethereum } = NETWORK_ADDRESSES;
+export const ARBITRUM_ONE_ACM = "0xD9dD18EB0cf10CbA837677f28A8F9Bda4bc2b157";
+export const ETHEREUM_ACM = "0x230058da2D23eb8836EC5DB7037ef7250c56E25E";
+export const OPBNBMAINNET_ACM = "0xA60Deae5344F1152426cA440fb6552eA0e3005D6";
+export const ARBITRUM_ONE_REWARD_DISTRIBUTORS = ["0x53b488baA4052094495b6De9E5505FE1Ee3EAc7a"];
+export const ETHEREUM_REWARD_DISTRIBUTORS = [
+  "0x134bfDEa7e68733921Bc6A87159FB0d68aBc6Cf8",
+  "0x76611EEA26aF8842281B56Bb742129E77133592F",
+  "0x886767B62C7ACD601672607373048FFD96Cf27B2",
+  "0x8473B767F68250F5309bae939337136a899E43F9",
+  "0x5f65A7b60b4F91229B8484F80bc2EEc52758EAf9",
+  "0x461dE281c453F447200D67C9Dd31b3046c8f49f8",
+  "0x7A91bEd36D96E4e644d3A181c287E0fcf9E9cc98",
+  "0xe72Aa7BaB160eaa2605964D2379AA56Cb4b9A1BB",
+  "0xDCB0CfA130496c749738Acbe2d6aA06C7C320f06",
+  "0x1e25CF968f12850003Db17E0Dba32108509C4359",
+];
+
+const vip356 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-356",
+    description: `### Description`,
+    forDescription: "I agree that Venus Protocol should proceed with this proposal",
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+  return makeProposal(
+    [
+      // Revoke Permissions
```

Comment:
Permissions on RewardDistributor contracts were set using the wildcard `0x0` (see it here, for [Arbitrum](https://github.com/VenusProtocol/vips/blob/main/multisig/proposals/arbitrumone/vip-004/index.ts#L104-L113) and [Ethereum](https://github.com/VenusProtocol/vips/blob/main/multisig/proposals/ethereum/vip-002/index.ts#L155-L164)). So, I suppose we should revoke the permission on the `0x0` address, not on specific addresses